core/admin/mailu/manage.py

```diff
@@ -121,7 +121,7 @@ def password(localpart, domain_name, password):
     email = f'{localpart}@{domain_name}'
     user  = models.User.query.get(email)
     if user:
-        user.set_password(password)
+        user.set_password(password, '')
```

Comment:
This change is not necessary

Reply:
I would like to the sessions for that user to be destroyed when the admin resets the password. Maybe I should change the interface to make it clearer

Reply:
clarified in 786da5e0